templates/options-radios.php

```diff
@@ -1,15 +1,28 @@
-<?php $cb_id = $this->get_element_id() . '-' . esc_attr( sanitize_text_field( $data_row['value'] ) ); ?>
+<?php
+/**
+ * Default template for Fieldmanager_Radios.
+ *
+ * @package Fieldmanager\Template
+ */
+
+$cb_id = $this->get_element_id() . '-' . esc_attr( sanitize_text_field( $data_row['value'] ) );
+?>
+
 <div class="fm-option">
 	<input
 		class="fm-element"
 		type="radio"
 		value="<?php echo esc_attr( $data_row['value'] ); ?>"
 		name="<?php echo esc_attr( $this->get_form_name() ); ?>"
 		id="<?php echo esc_attr( $cb_id ); ?>"
-		<?php echo $this->get_element_attributes(); ?>
-		<?php echo $this->option_selected( $data_row['value'], $value, "checked" ); ?>
+		<?php
+		echo $this->get_element_attributes(); // Escaped in fuction call WPCS: XSS okay.
+		?>
+		<?php
+		echo $this->option_selected( $data_row['value'], $value, 'checked' ); // Escaped in fuction call WPCS: XSS okay.
```

Comment:
This exception is described elsewhere in the PR as `// Escaped interally. xss ok.`, and also in `master` at https://github.com/alleyinteractive/wordpress-fieldmanager/blob/master/templates/options-checkboxes.php#L19. Again, not sure whether it's worth standardizing.